src/templates/.eslintrc.js

```diff
@@ -0,0 +1,54 @@
+const { resolve } = require('path')
+
+module.exports = {
+  root: true,
+
+  extends: [
+    'eslint:recommended',
+    'plugin:lodash-template/recommended-with-script'
```

Comment:
It seems like you're not adding this into any `plugins` array, may this be the problem

Reply:
It's not needed as the config adds the plugin automatically:
https://github.com/ota-meshi/eslint-plugin-lodash-template/blob/d1e00964657a8b67ebfdd8269a74628d078d3f3c/lib/configs/base.js#L7